byudml/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '0.6.8'
+__version__ = '0.6.9'
```

Comment:
I would wait to do this version bump until we are ready to merge devel into master

Reply:
Done.